packages/SwingSet/src/controller/controller.js

```diff
@@ -208,9 +208,8 @@ export async function makeSwingsetController(
     // Create an object with the property order that loadgen expects.
     const timedObj = { time: undefined, type: undefined, ...obj };
 
-    // Update the timedObj timestamp.
-    const timeMS = performance.timeOrigin + performance.now();
-    timedObj.time = timeMS / 1000;
+    // Update the timedObj timestamp, this uses unix gettimeofday()
+    timedObj.time = microtime.nowDouble();
```

Comment:
I would love if we could still keep a monotonic source in there. Is there any reason we couldn't include both `gettimeofday()` as `time` and `performance.now()` (without adding the origin, which becomes meaningless) as a second field?

```suggestion
    timedObj.time = microtime.nowDouble();
    timedObj.stamp = performance.now() / 1000;
```

That would require me to push an update to the loadgen, but that's pretty straightforward.

Reply:
yeah, ok, it feels like a bit of bloat to me, but I can see the value of it

Currently, when we want to compare slogfiles between two different validators, we have to pre-filter them with `jq 'delete(.time)'`, this will make that change to `delete(.time, .stamp)`. But actually my new timestamps-in-metering-results change will require something like `delete(.time, .stamp, .dr.[2].timestamps)`, so really we should build a specific tool for that comparison (which also tolerates only-partially-overlapping slogfiles, and startup-time replays, and such).

How about `.reltime` or `.relative` for the monotonic value? "stamp" doesn't speak to me as much.

Reply:
> really we should build a specific tool for that comparison (which also tolerates only-partially-overlapping slogfiles, and startup-time replays, and such).

Yes! Would love something that can stitch slogs together too! Currently my slog diffing is very limited:
https://github.com/Agoric/agoric-sdk/blob/1505285b137e1aa6a91246dc6f4b771650ef7612/scripts/process-integration-results.sh#L175-L184

> How about `.reltime` or `.relative` for the monotonic value? "stamp" doesn't speak to me as much.

Either is fine by me (I almost didn't want to include a name because I wasn't inspired)

Reply:
I pushed a change, it now uses `{ type, ...rest, time, monotime }`